src/wallet/api/wallet.cpp

```diff
@@ -1161,6 +1225,16 @@ AddressBook *WalletImpl::addressBook() const
     return m_addressBook;
 }
 
+Subaddress *WalletImpl::subaddress() const
```

Comment:
I really don't see the need for it (since the pointer value never changes inside the function), but I make the function (as well as other similar functions `history()` and `addressBook()`) non-const anyway according to your request. This doesn't cause any trouble for the compiler since those functions never get called in a context where `this` is const.